### PR TITLE
MAUI-860109: UG content for accessibility and keyboard support in ImageEditor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
**Description**
Add accessibility and keyboard navigation content for image editor control.

### CODE REVIEW

File: MAUI/ImageEditor/text.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 * [`IsEditable`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_IsEditable): Enables or disables text editing. The default value is `true`.
 * [`IsRotatable`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_IsRotatable): Enables or disables the text rotation. The default value is `true`.
 * [`TextAlignment`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_TextAlignment): Specifies the start, center, or end alignment of the text. The default value is `TextAlignment.Center`.
+* `Background`: Specifies the background color of the text annotation.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/text.md:51]: default value is transparent

Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/text.md:51]: image

Specify opacity is applied to background color.

File: MAUI/ImageEditor/text.md [hunk 2]
```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 
 ![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
 
+## Customize the background color of text annotations
+
+The .NET MAUI Image Editor supports customizing the background color of text annotations using `Background` property.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/text.md:94]: The background color of the text annotations can be customized by using the `Background` property.

File: MAUI/ImageEditor/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/accessibility.md:2]: add accessibility and keyboard under single md file

https://help.syncfusion.com/xamarin/scheduler/accessibility

File: MAUI/ImageEditor/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: Accessibility in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Accessibility in .NET MAUI Image Editor (SfImageEditor)
+
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/accessibility.md:11]: **remove**
The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) enables users to interact in different ways to help meet their visual, hearing, speech, dexterity, and cognitive needs.

File: MAUI/ImageEditor/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) enables users to interact in different ways to help meet their visual, hearing, speech, dexterity, and cognitive needs.
+
+The Image Editor can efficiently interact with toolbar icons and sliders, narrating their names and descriptions.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/accessibility.md:14]: The Image Editor is designed to work effectively with the icons and sliders on its toolbar, providing voice descriptions of their names and functions.

File: MAUI/ImageEditor/keyboard-shortcuts.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+title: Keyboard shortcuts in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using keyboard shortcuts in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Keyboard shortcuts in .NET MAUI Image Editor (SfImageEditor)
+
+The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) allows the users to navigate through Toolbar items using keyboard shortcuts in Windows and macOS platforms. The below table lists the available shortcuts and the corresponding operations.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/keyboard-shortcuts.md:12]: The [Syncfusion .NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) supports keyboard shortcuts for user interaction. The table provided outlines the various shortcuts and their associated functions.

File: MAUI/ImageEditor/keyboard-shortcuts.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+N> [ToolbarItemSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ToolbarItemSelected) event triggers for toolbar item tapped.
+
+### Toolbar
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/keyboard-shortcuts.md:21]: <b> Shortcut Keys </b>

File: MAUI/ImageEditor/keyboard-shortcuts.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Moves selection to the next (LTR) or Previous (RTL) focusable toolbar item.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/keyboard-shortcuts.md:25]: Moves selection to the next focusable toolbar item.

File: MAUI/ImageEditor/keyboard-shortcuts.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Moves selection to the next (LTR) or Previous (RTL) focusable toolbar item. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Moves selection to the previous (LTR) or next (RTL) focusable toolbar item.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/keyboard-shortcuts.md:30]: Moves selection to the previous focusable toolbar item.

File: MAUI/ImageEditor/keyboard-shortcuts.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + S<br/><br/></td><td>
+Save the current image. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + O<br/><br/></td><td>
+Opens a new image file for editing.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/keyboard-shortcuts.md:54]: Opens the image browser to import the new image inside ImageEditor.

File: MAUI/ImageEditor/text.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Updated default value

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Specified a opacity for background color

File: MAUI/ImageEditor/text.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Updated

File: MAUI/ImageEditor/accessibility.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/accessibility.md:2]: Added

File: MAUI/ImageEditor/accessibility.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/accessibility.md:11]: Removed

File: MAUI/ImageEditor/accessibility.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/accessibility.md]: Updated

File: MAUI/ImageEditor/keyboard-shortcuts.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/keyboard-shortcuts.md]: Updated

File: MAUI/ImageEditor/keyboard-shortcuts.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/keyboard-shortcuts.md]: Updated

File: MAUI/ImageEditor/keyboard-shortcuts.md (hunk 8 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/keyboard-shortcuts.md]: Updated

File: MAUI/ImageEditor/keyboard-shortcuts.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/keyboard-shortcuts.md]: Modified

File: MAUI/ImageEditor/keyboard-shortcuts.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/keyboard-shortcuts.md]: Modified

File: MAUI/ImageEditor/text.md [hunk 11]
```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
...
+           new ImageEditorTextSettings()
+           {
+               Background = Colors.white
+           });
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/text.md:122]: file name is imageeditor_background, change that to imageeditor-text-background-customization

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Changed

File: MAUI/ImageEditor/accessibility.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+title: Accessibility in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Accessibility in .NET MAUI Image Editor (SfImageEditor)
+
+The Image Editor is designed to work effectively with the icons and sliders on its toolbar, providing voice descriptions of their names and functions.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/accessibility.md:12]: The Image Editor is designed to work effectively with the icons and sliders on its toolbar, providing voice descriptions of their names, and functions.

File: MAUI/ImageEditor/accessibility.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+## Keyboard shortcuts in .NET MAUI Image Editor (SfImageEditor)
+
+The [Syncfusion .NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) supports keyboard shortcuts for user interaction. The table provided outlines the various shortcuts and their associated functions.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/accessibility.md:16]: The [Syncfusion .NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) supports keyboard shortcuts for user interaction. The table provided outlines various shortcuts and their associated functions.

File: MAUI/ImageEditor/text.md [hunk 14]
```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 
 ![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
 
+## Customize the background color of text annotations
+
+The background color of the text annotations can be customized by using the `Background` property.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/text.md:94]: The background color of the text annotations can be customized using the `Background` property.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

File: MAUI/ImageEditor/accessibility.md (hunk 12 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/accessibility.md]: Updated

File: MAUI/ImageEditor/accessibility.md (hunk 13 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/accessibility.md]: Updated

File: MAUI/ImageEditor/text.md (hunk 14 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Updated

File: MAUI/ImageEditor/text.md [hunk 15]
```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
...
 
+## Customize the background color of text annotations
+
+The background color of the text annotations can be customized using the `Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/text.md:100]: Mention Text color opacity code for programmatic annotation added

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md:100]: Mentioned

File: MAUI/ImageEditor/text.md [hunk 16]
```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
...
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnTextBackgroundClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.AddText("Good Day",
+           new ImageEditorTextSettings()
+           {
+               Background = Colors.White.WithAlpha(0.5f)
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/text.md:115]: Background = Colors.White.WithAlpha(0.5f)

no need to mention here, use like below

Background = Colors.White,
Opacity = 0.5

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/text.md]: Changed

**Review by karthikraja-arumugam — APPROVED** [review]

**Pull request merged**